src/client/app/heroic/heroic.module.ts

```diff
@@ -1,13 +1,13 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-import { FormsModule } from '@angular/forms';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 
 import { HeroicRoutingModule, routedComponents } from './heroic-routing.module';
 import { HeroDetailComponent } from './heroes/hero-detail.component';
 import { VillainDetailComponent } from './villains/villain-detail.component';
 
 @NgModule({
-  imports: [CommonModule, FormsModule, HeroicRoutingModule],
+  imports: [CommonModule, FormsModule, ReactiveFormsModule, HeroicRoutingModule],
```

Comment:
Per Kara, we should use Template-driven (FormsModule) or reactive (ReactiveFormsModule) but not both.

Reply:
yep